DarklyTests/Models/DataManagerTest.m

```diff
@@ -158,4 +160,54 @@ -(void)testAllEventsJsonData {
 
     XCTAssertTrue([eventUserKey isEqualToString: theUser.email]);    
 }
+
+
+-(void)testFindOrCreateUser {
+    NSString *userKey = @"thisisgus";
+    User *aUser = [[User alloc] init];
+    aUser.key = userKey;
+    aUser.email = @"gus@anemail.com";
+    aUser.updatedAt = [NSDate date];
+    aUser.config = user.config;
+        
+    [MTLManagedObjectAdapter managedObjectFromModel:aUser 
+                               insertingIntoContext: [[DataManager sharedManager]
+                                                      managedObjectContext] error: nil];
+    [[DataManager sharedManager] saveContext];
+    
+    XCTAssertNotNil(aUser);
+    
+    User *foundAgainUser = [[DataManager sharedManager] findUserWithkey: userKey];
+    
+    XCTAssertNotNil(foundAgainUser);
+    XCTAssertEqual(aUser.email, foundAgainUser.email);
+}
+
+-(void) testPurgeUsers {
+    NSDate *now = [NSDate date];
+    
+    for(int index = 0; index < 10; index++) {
```

Comment:
Should we move the kUserCacheSize from DataManager to somewhere accessible, and instead of 10 make this kUserCacheSize+1? if kUserCacheSize gets changed to 10 this test will break.